keps/sig-cluster-lifecycle/0015-kubeadm-join-control-plane.md

```diff
@@ -130,22 +128,17 @@ capabilities like e.g. kubeadm upgrade for HA clusters.
   The nodes must be created as a control plane instance or as workers and then are supposed to stick to the
   assigned role for their entire life cycle.
 
-- This proposal doesn't include a solution for etcd cluster management (but nothing in this proposal should
-  prevent to address this in future).
-
 - This proposal doesn't include a solution for API server load balancing (Nothing in this proposal
   should prevent users from choosing their preferred solution for API server load balancing).
 
-- This proposal doesn't address the ongoing discussion about kubeadm self-hosting; in light of
-  divide and conquer goal stated before, it is not planned to provide support for self-hosted clusters
-  neither in the initial proposal nor in the foreseeable future (but nothing in this proposal should
-  explicitly prevent to reconsider this in future as well).
+- This proposal doesn't include support for self-hosted clusters (but nothing in this proposal should
+  explicitly prevent us to reconsider this in the future as well).
 
 - This proposal doesn't provide an automated solution for transferring the CA key and other required
   certs from one control-plane instance to the other. More specifically, this proposal doesn't address
   the ongoing discussion about storage of kubeadm TLS assets in secrets and it is not planned
   to provide support for clusters with TLS stored in secrets (but nothing in this
-  proposal should explicitly prevent to reconsider this in future).
+  proposal should explicitly prevent us to reconsider this in the future as well).
```

Comment:
, but nothing in this proposal prevents us from reconsidering this in the future.